.pre-commit-config.yaml

```
@@ -50,7 +50,7 @@ repos:
         name: isort
         entry: isort
```

You're right, and besides this is needed too:

```suggestion
        entry: poetry run isort
```